### PR TITLE
Fix SA1133 handling of whitespace in code fix

### LINKED ISSUE #1883: SA1133 Code Fix incorrectly preserves whitespace
VS2015 Update 1 (RTM), StyleCop.Analyzers 1.0.0-RC.

Sample code:
```
#pragma warning disable SA1600
#pragma warning disable SA1649
#pragma warning disable SA1652

namespace SA1133CodeFix
{
    using System.ComponentModel;
    using System.Diagnostics.CodeAnalysis;

    [DefaultValue(true),
    SuppressMessage(null, null)]
    internal class Foo
    {
    }
}
```

Warning:

```Class1.cs(11,5,11,20): warning SA1133: Each attribute should be placed in its own set of square brackets.```

"Fixed" code:

```
#pragma warning disable SA1600
#pragma warning disable SA1649
#pragma warning disable SA1652

namespace SA1133CodeFix
{
    using System.ComponentModel;
    using System.Diagnostics.CodeAnalysis;

    [DefaultValue(true)]
    [    SuppressMessage(null, null)]
    internal class Foo
    {
    }
}

```

I think the code fix is preserving the whitespace between the start of the line and `SuppressMessage`.

### PR DESCRIPTION
**Pull request by nbarbettini**
Closes #1883

### CODE REVIEW

File: StyleCop.Analyzers/StyleCop.Analyzers.CodeFixes/ReadabilityRules/SA1133CodeFixProvider.cs [hunk 1]
```diff
@@ -77,7 +77,9 @@ private static List<AttributeListSyntax> GetNewAttributeList(AttributeListSyntax
 
             for (var i = 0; i < attributeList.Attributes.Count; i++)
             {
-                var newAttributes = SyntaxFactory.SingletonSeparatedList(attributeList.Attributes[i]);
+                var newAttributes = SyntaxFactory.SingletonSeparatedList(
+                    attributeList.Attributes[i].WithLeadingTrivia(
+                        attributeList.Attributes[i].GetLeadingTrivia().WithoutLeadingWhitespace()));
```
Comment (nbarbettini) [inline StyleCop.Analyzers/StyleCop.Analyzers.CodeFixes/ReadabilityRules/SA1133CodeFixProvider.cs]: :question: Is this an idiomatic way of stripping out the leading whitespace, or is there an easier/better way?

Comment (nbarbettini) [inline StyleCop.Analyzers/StyleCop.Analyzers.CodeFixes/ReadabilityRules/SA1133CodeFixProvider.cs]: :question: Is this an idiomatic way of removing leading whitespace from a symbol? Or is there an easier/better way?

Comment (vweijsters) [inline StyleCop.Analyzers/StyleCop.Analyzers.CodeFixes/ReadabilityRules/SA1133CodeFixProvider.cs]: I don't think there is another way. There is of course the possibility to add a new extension method to `TriviaHelper`.
Something with a syntax like this:
```csharp
internal static TSyntax WithoutLeadingWhitespace<TSyntax>(this TSyntax node, bool endOfLineIsWhitespace = true) where TSyntax : SyntaxNode
{
    ...
}
```

**Comment by vweijsters** [comment]
:+1: Looks good to me

**Pull request merged**